src/app/components/layout/side-menu/side-menu.component.ts

```diff
@@ -0,0 +1,28 @@
+import {Component, OnInit, AfterViewInit, Input} from '@angular/core';
+
+@Component({
+  selector: 'app-side-menu',
+  templateUrl: './side-menu.component.html',
+  styleUrls: ['./side-menu.component.scss']
+})
+export class SideMenuComponent implements OnInit, AfterViewInit {
+
+  @Input() sidenav;
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  ngAfterViewInit() {
+    const links = document.getElementsByClassName('link');
```

Comment:
Could you store this codes in some method like 'setOnClickListeners' and just execute the method in ngAfterViewInit? If we would like to add something to ngAfterViewInit this could be more readable if we only use methods here

Reply:
Good point. Done